cw/capture.py

```diff
@@ -17,12 +20,12 @@
 from chipwhisperer.common.traces import Trace
 from Crypto.Cipher import AES
 from Crypto.Hash import SHA3_256
-from cw_segmented import CwSegmented
-from pyXKCP import pyxkcp
 from tqdm import tqdm
-from waverunner import WaveRunner
 
+from cw_segmented import CwSegmented
+from pyXKCP import pyxkcp
 from util import device, plot
+from waverunner import WaveRunner
```

Comment:
Question is this change needed due to lint or is it a remainder from the rebasing maybe?

Reply:
I had a few problems with isort on my machine, which apparently kind of messed up the imports. I only realized that after I opened the PR, so I had to manually fix it.